app/models/subscriber.rb

```diff
@@ -0,0 +1,7 @@
+class Subscriber < ApplicationRecord
+  validates :email, uniqueness: { case_sensitive: false }, presence: true, email: { mode: :strict, require_fqdn: true }
+  validates :subscribed, inclusion: { in: [true, false], message: "must be true or false" }
+
+  self.implicit_order_column = "created_at"
```

Comment:
With UUIDs, this makes it so `Subscriber.first` and `Subscriber.last` always return the expected `Subscriber`